rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ Required ] As suggested by the previous reviewer, in the `initialize` method the `rental` should add itself to the `rentals` array in objects  `book` and `person`. Currently, your method is not implemented correctly, please see the example below for more details of how you can achieve this.

## Example code
```ruby
  def initialize(date, book, person)
    @date = date
    @book = book
    @person = person
    book.rentals << self
    person.rentals << self
  end
```